conn.go

```diff
@@ -330,6 +336,7 @@ func (l *Conn) processMessages() {
 				_, err := l.conn.Write(buf)
 				if err != nil {
 					l.Debug.Printf("Error Sending Message: %s", err.Error())
+					message.Channel <- &PacketResponse{Error: fmt.Errorf("unable to send request: %s", err)}
```

Comment:
Thanks, I've added a `close(message.Channel)` line below and moved the line which adds the channel to `l.chanResults` from before writing the packet (above) to after (below).